vanilla-framework/vanilla-framework
# Wrap the grid in a mixin so vanilla builds clean

## Done
Wrapped the grid in a mixin which in included in the vanilla mixin. Therefore, by including vanilla it will not produce any css. Also moved a few elements around to tidy up and make the file structure clearer.

## QA
_A lot of the changes are me tabbing the content as its wrapped in a mixin. I recommend code reviewing with `?w=1` in the URL_

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/grid/default/
- Check the grid is normal
- Now to test a blank include, run `sass scss/_vanilla.scss test.css`
- Then `cat test.css` or open in finder and ensure the output is empty

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1137
